CHANGELOG.md

```diff
@@ -3,6 +3,44 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v2.0.0 - [2025/10/14]
+
+### `Added`
+
+- [#124](https://github.com/nf-core/proteinfamilies/pull/124)
+  - Added new subworkflow `MERGE_FAMILIES` that can optionally merge similar (but not redundant) generated protein families. (by @vagkaratzas)
+  - Added new functionality to local module `IDENTIFY_REDUNDANT_IDS` which now also identifies similar family ids, along with their pairwise similarity scores. (by @vagkaratzas)
+  - Added new local module `POOL_SIMILAR_COMPONENTS` that generates family clusters, from a family-similarity edgelist. (by @vagkaratzas)
+  - Added new local module `MERGE_SEEDS` that merges seed alignments of similar families, before restarting the family generation subworkflow. (by @vagkaratzas)
+- [#118](https://github.com/nf-core/proteinfamilies/pull/118)
+  - Added preprint citation to the repo. (by @vagkaratzas)
+  - Added separate metro map files for dark and light browser modes. (by @vagkaratzas)
+  - Added new local module `EXTRACT_FAMILY_MEMBERS` that outputs a 2-col TSV file with final family identifiers and all their member sequence identifiers. (by @vagkaratzas)
```

Comment:
Say in which directory one can find the file and what it's called? (I understand this is in the output.md, but makes it easy for the curious.)